lib/modules/datasource/puppet-forge/index.spec.ts

```diff
@@ -0,0 +1,219 @@
+import { getPkgReleases } from '..';
+import { Fixtures } from '../../../../test/fixtures';
+import * as httpMock from '../../../../test/http-mock';
+import { PuppetForgeDatasource } from '.';
+
+const puppetforgeReleases = Fixtures.get('puppetforge-response.json');
+
+const datasource = PuppetForgeDatasource.id;
+
+describe('modules/datasource/puppet-forge/index', () => {
+  describe('getReleases', () => {
+    it('should use default forge if no other provided', async () => {
+      httpMock
+        .scope('https://forgeapi.puppet.com')
+        .get('/v3/modules/puppetlabs-apache')
+        .query({ exclude_fields: 'current_release' })
+        .reply(200, puppetforgeReleases);
+
+      const res = await getPkgReleases({
+        datasource,
+        depName: 'puppetlabs/apache',
+        packageName: 'puppetlabs/apache',
+      });
+      expect(res).toBeDefined();
+      expect(res.registryUrl).toBe('https://forgeapi.puppet.com');
```

Comment:
please use `toMatchObject` and also validate other fields of response like number of versions